README.md

```diff
@@ -15,6 +15,9 @@ Java bindings to the [tree-sitter] parsing library.
 git clone https://github.com/tree-sitter/java-tree-sitter
 cd java-tree-sitter
 git submodule init
+git submodule update
+# build tree-sitter and tree-sitter-java native libraries,
+# for example by replicating the steps in .github/workflows/ci.yml
```

Comment:
```suggestion
```
Add this to the bullet points instead:
```markdown
- Install the `tree-sitter` & `tree-sitter-java` libraries
````

Reply:
Done. FYI I couldn't figure out how to "Install the tree-sitter & tree-sitter-java libraries" other than to replicate `.github/workflows/ci.yml` locally.

The Windows build I couldn't get to work at all. Despite using the exact same version of jextract, it would generate slightly different/incompatible output, causing compile errors in `Query.java` and `Parser.java`.

Reply:
I've fixed both of those now but some tests still crash.